api/swagger.yaml

```diff
@@ -868,26 +868,227 @@ definitions:
         items:
           type: "string"
 
-  NetworkConfig:
-    description: "TODO: check is correct"
+  NetworkSettings:
+    description: "NetworkSettings exposes the network settings in the api"
     type: "object"
     properties:
       Bridge:
+        description: |
+          Bridge is the Bridge name the network uses (for example, `docker0`).
```

Comment:
I took these from the source code; They are [GoDoc style descriptions](https://blog.golang.org/godoc-documenting-go-code); ultimately, the Go code will be generated from this Swagger file, in which case we should probably stick with that convention (otherwise CI will fail)